src/libYARP_OS/src/DgramTwoWayStream.cpp

```diff
@@ -38,8 +38,7 @@ using namespace yarp::os::impl;
 using namespace yarp::os;
 
 #define CRC_SIZE 8
-#define READ_SIZE (120000-CRC_SIZE)
-#define WRITE_SIZE (60000-CRC_SIZE)
+#define UDP_MAX_DATAGRAM_SIZE 65507 - CRC_SIZE
```

Comment:
Here: https://stackoverflow.com/questions/1098897/what-is-the-largest-safe-udp-packet-size-on-the-internet is explained why 65507.